frontend/src/pages/AccountSend.vue

```diff
@@ -279,12 +517,20 @@ import {
 } from 'src/utils/ethers';
 import { humanizeTokenAmount, humanizeMinSendAmount, humanizeArithmeticResult } from 'src/utils/utils';
 import { generatePaymentLink, parsePaymentLink } from 'src/utils/payment-links';
+import { SendBatch } from 'app/../umbra-js/build/src/types';
```

Comment:
We should avoid relative paths for imports and be consistent with the other umbra-js imports, by only importing things exported by the package. So if the below doesn't work we should update umbra-js to export the types. In other words, the current import syntax only works for us in a monorepo, the below import syntax would work for anyone
```suggestion
import { SendBatch } from '@umbracash/umbra-js';
```

Reply:
Makes sense, thank you! Fixed it here: https://github.com/ScopeLift/umbra-protocol/pull/491/commits/4ba9948658c97f5f08836e3701b9ce46eeaf7385